exec/java-exec/src/main/java/org/apache/drill/exec/util/Utilities.java

```diff
@@ -60,9 +60,12 @@ public static String getFileNameForQueryFragment(FragmentContext context, String
    * QueryContextInformation is derived from the current state of the process.
    *
    * @param defaultSchemaName
+   * @param sessionId
+   * @param manager
    * @return A {@link org.apache.drill.exec.proto.BitControl.QueryContextInformation} with given <i>defaultSchemaName</i>.
    */
-  public static QueryContextInformation createQueryContextInfo(final String defaultSchemaName, final String sessionId) {
+  public static QueryContextInformation createQueryContextInfo(final String defaultSchemaName,
+      final String sessionId, final OptionManager manager) {
```

Comment:
The OptionManager is not used in this constructor ?

Reply:
Done